apps/www/components/Frame/Popover/UserNav.js

```diff
@@ -173,29 +173,7 @@ const UserNav = ({
                   />
                   <div {...styles.navSection}>
                     <div {...styles.navLinks}>
-                      {me?.activeMembership &&
-                      ['ABO', 'BENEFACTOR_ABO'].includes(
-                        me.activeMembership?.type?.name,
-                      ) ? (
-                        <NavLink
-                          href='/verstaerkung-holen'
-                          currentPath={currentPath}
-                          closeHandler={closeHandler}
-                          large
-                        >
-                          {t('nav/invite-friends')}
-                        </NavLink>
-                      ) : me.accessCampaigns.length > 0 ? (
-                        <NavLink
-                          href='/teilen'
-                          currentPath={currentPath}
-                          closeHandler={closeHandler}
-                          large
-                        >
-                          {t('nav/share')}
-                        </NavLink>
-                      ) : null}
-                      {!me?.activeMembership && me.accessCampaigns.length > 0 && (
+                      {me.accessCampaigns.length > 0 && (
```

Comment:
```suggestion
                      {me?.accessCampaigns?.length > 0 && (
```

(me could be falsy, `accessCamapaings` could be null)